src/dynamics/ergodic/ergodic.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2022 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import dynamics.ergodic.measure_preserving
+
+/-!
+# Ergodic maps and measures
+
+Let `f : α → α` be measure preserving with respect to a measure `μ`. We say `f` is ergodic with
+respect to `μ` (or `μ` is ergodic with respect to `f`) if the only measurable sets `s` such that
+`f⁻¹' s = s` are either almost empty or full.
+
+In this file we define ergodic maps / measures together with quasi-ergodic maps / measures and
+provide some basic API.
```

Comment:
Can you add a sentence to explain what quasi-ergodic means? The file docstring explains only ergodic and the reader is not told that the quasi refers to quasi-measure-preserving.

Reply:
Good point, done.